stellargraph/mapper/full_batch_generators.py

```diff
@@ -123,7 +123,7 @@ def __init__(
             else:
                 raise ValueError("argument 'transform' must be a callable.")
 
-        elif self.method in ["gcn", "chebyshev", "sgc"]:
+        elif self.method in ["gcn", "sgc"]:
```

Comment:
I assume that it's impossible for anyone to have actually used `chebyshev` here? As in, if they passed in `"chebyshev"` their code would've hit an error like #411, and so it's not worth doing a special case like `GCN_Aadj_feats_op`?

Reply:
It may have been possible to create the generator object, but it wouldn't have been useable in any way, since the shape of `generator.features` would've been wrong for anything downstream. Here we do already have an `else: raise` at the bottom too so it's probably fine?